source/extensions/filters/network/ext_authz/ext_authz.cc

```diff
@@ -78,6 +78,10 @@ void Filter::onComplete(Filters::Common::ExtAuthz::CheckStatus status) {
     config_->stats().cx_closed_.inc();
     filter_callbacks_->connection().close(Network::ConnectionCloseType::NoFlush);
   } else {
+    if (status != Filters::Common::ExtAuthz::CheckStatus::OK && config_->failureModeAllow()) {
```

Comment:
Can the check be for the explicit error status(Denied and Error) incase other enums are added?

Reply:
good point. will do so.